config/staging.js

```diff
@@ -16,10 +16,6 @@ module.exports = {
     },
   },
 
-  cors: {
```

Comment:
We are not overriding the values in env specific config. So it shouldnt be added here... 🤔

Reply:
Okay, this has been removed 👍